### DISCUSSION TITLE
how to annotate intraoral 3d Scans in stl file foramt

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [facebookresearch/pytorch3d#1746] [Q&A]
I have a data folder contain real life intra-oral 3D scans from multiple subjects in multiple positions.
In the validation data folder can be find real life 3D scans but the rotation was made by blender (software) using multiple artificial axes and rotation angles.
This validation data can be used for validation any kind of rotation axes finder algorithm.
The axes and the rotation angles can be find in the algo_prec.csv file for each artificial rotations.
how to prepare them for Pytorch3D models! 
thanks in advance

**Comment by bottler** [comment]
I think you need to do this yourself in code: Load the viewpoints and the data separately and transform the data.

**Reply by abdelkareemkobo** [reply]
I am totally new to 3d, can you help more ! 
what libs should i start look for. 
references or any examples i can use. 
is there any tools i could try. 
thanks a lot for quick response
